docs/extend/extend-apiml/api-mediation-personal-access-token.md

```diff
@@ -0,0 +1,151 @@
+# Personal Access Token
+
+The API ML allows generating, validating and invalidating a **Personal Access Token (PAT)** that can be used, for instance, to leverage modern tools such as Version Control Systems (VCS) during
+the development of some application on the z/OS system.
+
+To do that, the API ML offers a set of REST APIs to:
+
+1. [Generate the token](#generate-of-the-token)
+2. [Validate the token](#validate-the-token)
+3. [Invalidate a specific token](#invalidate-a-specific-token)
+4. [Invalidate all the tokens for a user](#invalidate-all-the-tokens-for-a-user)
+5. [Invalidate all the tokens for a service](#invalidate-all-the-tokens-for-a-service)
+6. [Retrieve all the tokens and rules](#retrieve-all-the-tokens-and-rules)
+7. [Evict the non-relevant tokens and rules](#evict-the-non-relevant-tokens-and-rules)
+
+The **User** is able to :
+* Generate the token
+* Validate the token
+* Invalidate a specific token
+* Invalidate all tokens
+
+The **Security Administrator** is able to:
+* Invalidate the token for a user
+* Invalidate all the tokens for a user
+* Invalidate all tokens
+* Evict invalidated tokens and rules which are not relevant anymore
+
+## Generate of the token
+
+The user can create the Personal Access Token by calling the following REST API endpoint via Gateway:
+
+`POST /auth/access-token/generate`
+The full path of the `/auth/access-token/generate` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/generate`.
+
+The request requires the body in the following format:
+
+```json
+{
+    "validity": 90,
+    "scopes": ["<serviceId_1>,<serviceId_2>"]
+}
+```
+
+The **validity** refers to the expiration time of the tokens. The maximum threshold is 90 days.
+The **scopes** allows to limit the access on a service level, so it will introduce higher level of security in some aspects. Users will be forced to provide a scope and if no service is specified, it will work for none.
+
+When creation is successful, the response to the request is a body containing the PAT. When creation fails, the user receives a 401 status code.
```

Comment:
It would be good to show an example of the successful and unsuccessful response in this case.